tests/acceptance/EA04OrderCest.php

```diff
@@ -252,4 +252,31 @@ public function order_受注登録(\AcceptanceTester $I)
         $I->see('受注情報を保存しました。', OrderEditPage::$登録完了メッセージ);
     }
 
+    public function order_ー括受注のステータス変更(\AcceptanceTester $I)
+    {
+        $I->wantTo('EA0402-UC02-T02_ー括受注のステータス変更');
+
+        $findOrders = Fixtures::get('findOrders'); // Closure
+
+        $NewOrders = array_filter($findOrders(), function ($Order) {
+            return $Order->getOrderStatus()->getId() == OrderStatus::NEW;
+        });
+        OrderManagePage::go($I)->受注ステータス検索(OrderStatus::NEW);
+        $I->see('検索結果：'.count($NewOrders).'件が該当しました', OrderManagePage::$検索結果_メッセージ);
+
+        $DeliveredOrders = array_filter($findOrders(), function ($Order) {
+            return $Order->getOrderStatus()->getId() == OrderStatus::DELIVERED;
+        });
+        OrderManagePage::go($I)->受注ステータス検索(OrderStatus::DELIVERED);
+        $I->see('検索結果：'.count($DeliveredOrders).'件が該当しました', OrderManagePage::$検索結果_メッセージ);
+
+        OrderManagePage::go($I)->受注ステータス検索(OrderStatus::NEW)
+            ->一覧_全選択()
+            ->受注ステータス変更('発送済み');
+
+        $I->wait(10);
```

Comment:
why need this wait ?

Reply:
Hi @ryo-endo, I added it while testing, to make sure all orders which is updated status be done before search them again. But I'm realize no need wait due to codeception was waited as default behavior. Then I removed it.